### PR TITLE
Gmiguel

### PR DESCRIPTION
**Pull request by gracemiguel**

### CODE REVIEW

File: test/seeds/test17.ged [hunk 1]
```diff
@@ -0,0 +1,257 @@
+0 HEAD
```
Comment (samanthainneo99) [inline test/seeds/test17.ged:1]: Just make sure that this doesn't break other people's testing if they used a test17.ged as well

**Review by samanthainneo99 — APPROVED** [review]
I think it's fine but check the test17.ged doesn't mess with the test_suite

File: test/us06_test.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+
+
+
+class us06_test(unittest.TestCase):
+    def test1(self):
+        filename = "us06/test1.ged"
+        s = "The following people have divorces after their death date['DANIEL SOLLER']"
+        self.assertEqual(s, us06.us06(filename))
```
Comment (ming1in) [inline test/us06_test.py]: This test is currently failing

File: test/us06_test.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import os
+import sys
+sys.path.append(os.path.abspath('../src'))
+import us06
+
+
+
+
+
+class us06_test(unittest.TestCase):
```
Comment (ming1in) [inline test/us06_test.py:10]: add this test to `test_suite.py`

File: src/us05.py [hunk 4]
```diff
@@ -28,3 +28,4 @@ def us05(gedcom_file):
         print("The following people have marriage dates after death date: " + str(error))
 
 
+us05("test15.ged")
```
Comment (ming1in) [inline src/us05.py]: should not have this here. This will unnecessarily trigger every time this file is called

**Review by ming1in — APPROVED** [review]
looks good, should try to merge this soon because ya branch is starting to having a lot of unmerged code. Before you merge you should remove the `src/__pycache__` files because they're unneeded python cache files

**Pull request merged**